core.js

```diff
@@ -3267,11 +3268,14 @@ const _runJavascript = (jsString, window, filename = 'script', lineOffset = 0, c
 };
 
 let rafCbs = [];
-const tickAnimationFrame = () => {
+const tickAnimationFrame = which => {
   if (rafCbs.length > 0) {
     const localRafCbs = rafCbs.slice();
 
     const _handleRaf = localRafCb => {
+      if (which && !which(localRafCb[whichSymbol])) {
```

Comment:
`which` should probably never be false, so there's no point to check.

Reply:
Also, I would make `which` a constant to compare against, to optimize. This is one of the hottest paths in the codebase.